docs/clients/iquart_flight_controller_interface.rst

```diff
@@ -87,12 +87,13 @@ A minimal working example for the IQUartFlightControllerInterfaceClient is:
 
 .. code-block:: Matlab
 
-    % Make a communication interface object
-    com = MessageInterface(’COM18’,115200);
-    % Make a IQUartFlightControllerInterfaceClient object with obj_id 0
-    iquartFlightControllerInterface = IQUartFlightControllerInterfaceClient(’com’,com);
-    % Use the IQUartFlightControllerInterfaceClient object
-    pulsingVoltageMode = iquartFlightControllerInterface.get(’telemetry’);
+%   % Make a communication interface object
```

Comment:
it's hard to tell from the github version, but do you mean to have these all double commented/the code commented? if people try to copy and paste, then they're just going to get a bunch of commented code